src/components/SpaceSidebarFooter.vue

```diff
@@ -0,0 +1,62 @@
+<script setup>
+defineProps({
+  space: Object
+});
+</script>
+
+<script>
+export default {
+  data () {
+    return {
+      socials: [
+        {
+          url: 'https://github.com/',
+          spaceAttribute: 'github',
+          icon: 'github',
+        },
+        {
+          url: 'https://twitter.com/',
+          spaceAttribute: 'twitter',
+          icon: 'twitter',
+        },
+        {
+          url: 'https://discord.gg/',
+          spaceAttribute: 'discord',
+          icon: 'discord',
+        },
+      ]
+    }
+  },
+  computed: {
+    spaceSocials() {
+      return this.socials.filter(social => this.space[social.spaceAttribute])
+    }
+  }
+}
+</script>
+
+<template>
+  <div
+    v-if="space && spaceSocials.length"
```

Comment:
`space` is always available when this component is mounted. We should add a skeleton loading for this footer to the `SpaceSidebarSkeleton` component.